ui/packages/docs/src/components/Params/character_data.json

```diff
@@ -464,6 +464,18 @@
       "desc": "0 for Tap (default), value between 1 and 300 for Hold. The number is added to the minimum holding frames for Tap to turn into Hold (16)."
     }
   ],
+  "navia": [
+    {
+      "ability": "skill",
+      "param": "hold",
+      "desc": "0 for Tap (default), value between 1 and 241 for Hold. The number is added to the minimum holding frames for Tap to turn into Hold (41), subtracted by 1 to account for requiring a 1 to activate"
+    },
+    {
+      "ability": "skill",
+      "param": "shrapnel",
+      "desc": "The number of shrapnel use the skill with. 0 (default) uses the number acquired throughout the config, 1-6 force that many shrapnel."
+    }
```

Comment:
remove, because I removed this param (no forcing of something that can be handled by the user).